HIP/hip-32.md

```diff
@@ -57,7 +57,9 @@ An automatically-created account will be created with only one public key (the o
 
 At most one account can ever have a given alias. If a transaction auto-creates the account, any further transfers to that alias will simply be deposited in that account, without creating anything, and with no creation fee being charged.
 
-An account created normally has no alias. It can be given an alias with an account update, but only if that alias is not currently used by any other account, and only if the update transaction is signed by the private key corresponding to the public key in the alias.
+An account created normally has no alias. ~~It can be given an alias with an account update, but only if that alias is not currently used by any other account, and only if the update transaction is signed by the private key corresponding to the public key in the alias.~~
+* **UPDATE:** This ability has been removed by [HIP-583](https://hips.hedera.com/hip/hip-583#update-account-with-alias),
```

Comment:
Should we be clear that it was never implemented but the goal was rather given up on after exploring HIP 583 and 631?
```suggestion
* **UPDATE:** This intent has been revisited and retracted by [HIP-583](https://hips.hedera.com/hip/hip-583#update-account-with-alias),
```

Reply:
TY I like your wording.